### ISSUE TITLE
0 in created field

### ISSUE THREAD

**Issue by mdlincoln** [cooperhewitt/the-pen-data#2]
There are a handful of 0's in the `created` field that will cause some UNIX timestamp processing commands to choke.

``` r
     id tool_id bundle_id refers_to_object_id created
  (int)   (int)     (chr)               (int)   (chr)
1   112       0      6o16            18112325       0
2   121       0      6o16            18439365       0
3   126       0      6o16            18489509       0
```

**Comment by micahwalter** [comment]
Yep, these are also most likely from testing things early on. I'll try and prune them out.

**Comment by micahwalter** [comment]
OK, I've created a new version of the dataset here 54fbe816145274402d0c7cc403c8ec741bc39e20 and have eliminated rows where the created column is 0. Please have a look and let me know if it worked and then I'll merge it in.

**Comment by mdlincoln** [comment]
``` r
> any(pen_raw$created == 0)
[1] FALSE
```

:+1: all fixed!

**Comment by micahwalter** [comment]
Thanks! Merged in 54fbe816145274402d0c7cc403c8ec741bc39e20 

Does this also resolve #1 ?
